From 7e89372ddd2b776f71c6ea45d208defb9fa54c5e Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 20 Sep 2022 15:43:02 -0700
Subject: [PATCH] Enable auto-probe for dg1, dg2, ats-m

---

This should theoretically allow auto-probe of DG1, DG2, and ATS-M
graphics devices without a command-line option.


 drivers/gpu/drm/i915/i915_pci.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_pci.c b/drivers/gpu/drm/i915/i915_pci.c
index aacc10f2e73f..8b7c89a2d097 100644
--- a/drivers/gpu/drm/i915/i915_pci.c
+++ b/drivers/gpu/drm/i915/i915_pci.c
@@ -921,7 +921,7 @@ static const struct intel_device_info dg1_info = {
 	.graphics.rel = 10,
 	PLATFORM(INTEL_DG1),
 	.display.pipe_mask = BIT(PIPE_A) | BIT(PIPE_B) | BIT(PIPE_C) | BIT(PIPE_D),
-	.require_force_probe = 1,
+	.require_force_probe = 0,
 	.platform_engine_mask =
 		BIT(RCS0) | BIT(BCS0) | BIT(VECS0) |
 		BIT(VCS0) | BIT(VCS2),
@@ -1072,13 +1072,13 @@ static const struct intel_device_info dg2_info = {
 	XE_LPD_FEATURES,
 	.display.cpu_transcoder_mask = BIT(TRANSCODER_A) | BIT(TRANSCODER_B) |
 			       BIT(TRANSCODER_C) | BIT(TRANSCODER_D),
-	.require_force_probe = 1,
+	.require_force_probe = 0,
 };
 
 static const struct intel_device_info ats_m_info = {
 	DG2_FEATURES,
 	.display = { 0 },
-	.require_force_probe = 1,
+	.require_force_probe = 0,
 };
 
 #define XE_HPC_FEATURES \
-- 
2.37.3

